### PR TITLE
Feat/dbrestructure

### PR DESCRIPTION
**Pull request by henrimikaelt**

### CODE REVIEW

File: db/init.sql [hunk 1]
```diff
@@ -94,52 +104,118 @@ prevent confusion about were they left out intentionally or by accident
...
+    (13, 'Recital: New Generations - Astrid Bjelland', 'Resitaali: New Generations - Astrid Bjelland', 'sun2eng.txt', 'sun2fin.txt'),
+    (14, 'Masterclass: Sébastian Jacot', 'Mestarikurssi: Sébastian Jacot', 'sun3eng.txt', 'sun3fin.txt'),
+    (15, 'Recital: La flute enchantée - Sébastian Jacot', 'Resitaali: La flute enchantée - Sébastian Jacot', 'sun4eng.txt', 'sun4fin.txt'),
+    (16, 'Panel Discussion: In the Spotlight!', 'Paneelikeskustelu: Valokeilassa!', 'sun5eng.txt', 'sun5fin.txt'),
+    (17, 'JÄÄ//ICE Gala Concert', 'JÄÄ//ICE - Gaalakonsertti', 'sun6eng.txt', 'sun6fin.txt');
 
-INSERT INTO programme (concertid, artistid, compositionid, performancetime)
+--concerts data
+
+INSERT INTO concert (concertid, concertname, concertdate, consertstarttime, concertinfo_eng, concertinfo_fin, venueid)
```
Comment (ElianaJu) [inline db/init.sql:160]: column concertendtime missing. Also, the name of the column consertstarttime is correct but there is a typo (consert insteed of concert like all the other names)

Comment (Kronkeli) [reply db/init.sql:160]: Nice looking review with the relevant part of code in it :D

Comment (Kronkeli) [reply db/init.sql:160]: But how did that not fail in the automatic testing? It makes the whole database structure and runs the init.sql.

File: db/init.sql [hunk 2]
```diff
@@ -94,52 +104,118 @@ prevent confusion about were they left out intentionally or by accident
...
+    (7, 'www.beatrizmacias.com', 'https://www.facebook.com/beatriz.macias.58152/', 'https://www.youtube.com/channel/UCeCGeIztVZ-UNr_NXhox1hQ', 'https://www.instagram.com/beatrizmaciasflutist/', NULL, 7),
+    (8, NULL,'https://www.facebook.com/jenny.villanen' , NULL, 'https://www.instagram.com/jennyvillanen/', NULL, 8),
+    (9, 'https://www.mallavivolin.com/', 'https://www.facebook.com/kuutiomeduusa', 'https://www.youtube.com/watch?v=BC1lpARg04w','https://www.instagram.com/jumihiutale/', NULL , 9),
+    (10, 'http://www.evaalkula.com/musiikki-1', 'https://www.facebook.com/alkulaeva', 'https://www.youtube.com/watch?v=q6IbBFU1zgE','https://www.facebook.com/alkulaeva/', 'https://open.spotify.com/artist/6Bs5O9Ssp2A6JHxhu6nV24 ', 10),
+    (11, NULL, 'https://www.facebook.com/hannaleena.savolainen', NULL, NULL, NULL, 11),
+    (12, NULL, 'https://www.facebook.com/Piktor4ik','https://www.youtube.com/watch?v=dce0D3bEj9M','https://www.instagram.com/viktor_pellia/?hl=en', NULL, 12);
+
```
Comment (ElianaJu) [inline db/init.sql:119]: still missing the values of the photo table

File: db/init.sql [hunk 3]
```diff
@@ -1,7 +1,5 @@
---tables
-
 CREATE TABLE IF NOT EXISTS artist (
-    artistid SERIAL,
+    artistid INT NOT NULL,
     firstname VARCHAR(25) NOT NULL,
```
Comment (Kronkeli) [inline db/init.sql:3]: I'm a bit worried, that the database ("tffgodb") isn't defined in the beginnnig. I think in development it works fine with the docker-compose telling the database name in the environment variables, but we won't have them necessarily in the cloud hosting service. So as an insucrance I would add the connecting to the database with
`\c tffgodb`

**Review by ElianaJu — COMMENTED** [review]
Just the issue with the column concertendtime but otherwise, the rest is good. So we can merge and you can change that later or we can wait for the change and then merge

File: db/init.sql (hunk 1 above)
Comment (ElianaJu) [reply db/init.sql:160]: good question XD we don't call concert in a query so maybe we don't have any errors for now ?

**Comment by henrimikaelt** [comment]
In case you're wondering, the first connect is so that when in local environment the default db is already tffgo, people don't have to change their local variables and a db cannot be dropped unless connected to some other db first

**Pull request merged**